src/cdk/drag-drop/preview-ref.ts

```diff
@@ -169,15 +137,18 @@ export class PreviewRef {
         'pointer-events': 'none',
         // We have to reset the margin, because it can throw off positioning relative to the viewport.
         'margin': '0',
-        'position': 'absolute',
+        'position': 'fixed',
         'top': '0',
         'left': '0',
+        'z-index': this._zIndex + '',
```

Comment:
do we need z-index if its in a popover?

Reply:
We still have it, because there's a public API to set it and it case we're running on a browser that doesn't support popovers.

Reply:
I think all the browsers we support also support popover, but I guess we can leave it until we deprecate the zindex API